### ISSUE TITLE
static (index.html etc) not working

### ISSUE THREAD

**Issue by CnApTaK** [metarhia/metarhia.com#105]
11:10:45  W1   info    Listen port 80 in worker 1
11:10:45  W2   info    Listen port 443 in worker 2
11:10:45  W1   info    Application started in worker 1
11:10:45  W2   info    Application started in worker 2
...here i open http://127.0.0.1/ in chrome...
11:10:46  W2   error   127.0.0.1        GET     /       404     Not Found

**Comment by tshemsedinov** [comment]
See this video: https://youtu.be/Y2LYPuQwXQs
And if you can't reproduce it, please provide your video. @CnApTaK

**Comment by CnApTaK** [comment]
https://www.youtube.com/watch?v=OcQssx1r9Yo

**Comment by CnApTaK** [comment]
...same thing with other files in static folder...
2021-01-25T13:24:42.336Z [error] 127.0.0.1	GET	/	404	Not Found
2021-01-25T13:24:58.142Z [error] 127.0.0.1	GET	/console.css	404	Not Found
2021-01-25T13:25:14.631Z [error] 127.0.0.1	GET	/favicon.png	404	Not Found
2021-01-25T13:25:31.309Z [error] 127.0.0.1	GET	/worker.js	404	Not Found
2021-01-25T13:25:40.418Z [error] 127.0.0.1	GET	/metacom.js	404	Not Found

**Comment by CnApTaK** [comment]
this is kinda solves the problem on windows. add two lines in node_modules\impress\lib\application.js :
'use strict';

```
const { node, npm, metarhia } = require('./dependencies.js');
const { path, events, fs, fsp } = node;
const { metautil, metavm } = metarhia;

// workaround for windows
const path_join_original = path.join;
path.join = (a, b) => path_join_original.join(a, b).replaceAll('\\','/');
```

**Comment by tshemsedinov** [comment]
If you have time to debug please contact me in telegram.

**Comment by tshemsedinov** [comment]
@CnApTaK fixed in https://github.com/metarhia/impress/pull/1442 and will be published ASAP. We need to test it before.

**Comment by tshemsedinov** [comment]
It works and published as impress 2.0.2
FYI @CnApTaK
